svn/common.py

```diff
@@ -49,10 +51,13 @@ def run_command(self, subcommand, args, success_code=0,
 
         _logger.debug("RUN: %s" % (cmd,))
 
+        environment_variables = copy.deepcopy(os.environ)
```

Comment:
Umm.. What if a person's env variables contain nested dictionary? I can change it but it's better to take deepcopy into account